code_to_gast/expression/py_expression.py

```diff
@@ -28,6 +28,15 @@ def call_to_gast(node):
 handles attributes for python expressions
 """
 def attribute_to_gast(node):
-    gast = {"type": "attribute", "id": node.attr}
-    gast["value"] = pr.node_to_gast(node.value)
+    gast = {"value": pr.node_to_gast(node.value)} 
+
+    if node.attr == "append":
```

Comment:
Nit: I find it easier to reason about when functions return early instead of having "nested" conditionals.

```py
if node.attr == "append":
    ...
    return gast

if node.attr == "pop":
    ...
    return gast

...
```

Reply:
Okay I can update that.